src/sentry/tagstore/v2/backend.py

```diff
@@ -459,7 +459,7 @@ def get_group_tag_value_count(self, project_id, group_id, environment_id, key):
                     SELECT times_seen as t
                     FROM tagstore_grouptagvalue
                     INNER JOIN tagstore_tagkey
-                    ON (tagstore_grouptagvalue.key_id = tagstore_tagkey.id)
+                    ON (tagstore_grouptagvalue.key = tagstore_tagkey.id)
```

Comment:
I thought you were migrating this to `key_id`?

Reply:
I'd love to figure out how. I did previously (see commits, not sure how to find Travis history) and I got very weird errors. Trying to get everything to pass first.

Reply:
I mean, worst case, we create migration to drop the tables, and another migration to create them again. :)